client/src/pages/EditProfile.js

```diff
@@ -133,13 +156,18 @@ class EditProfilePage extends Component {
                   <h1>Edit Profile</h1>
                 </Grid>
                 <Grid item xs={12}>
-                  <form noValidate autoComplete="off" method="POST">
+                  <form noValidate autoComplete="off" method="POST" onSubmit={this.handleSubmit}>
                     <Grid container spacing={3} className="pb-1">
                       <Grid item xs={1}></Grid>
                       <Grid item xs={9}>
-                        <Grid container spacing={3}>
+                        <Grid container spacing={3}>  
                           <Grid item xs={5} className="text-right">
                             <p>FIRST NAME</p>
+                            {
+                              (this.state.errors)
+                              ? <div style={{ color: "red" }}>{this.state.errors.firstName}</div>
+                              : <div></div>
```

Comment:
is an empty `div` required if there are no errors?

Reply:
I don't need an empty div! I changed it to null